packages/apps/src/initSettings.ts

```diff
@@ -24,6 +24,20 @@ function getApiUrl (): string {
 
     assert(url.startsWith('ws://') || url.startsWith('wss://'), 'Non-prefixed ws/wss url');
 
+    const urlArray = ['127.0.0.1', 'localhost', '.cere.network'];
+    let hostname = (new URL(url)).hostname;
+    const port = Number(new URL(url).port);
+
+    if (port) {
+      assert(port < 64000, 'Invalid ws provider');
+    }
+
+    if (hostname !== 'localhost') {
+      hostname = hostname.substr(-13);
```

Comment:
The code is not clear. What is `-13`? Move this logic to a separate function with clear name.

Reply:
Done